backend/docker/prod/ecs/Dockerfile

```diff
@@ -53,13 +53,10 @@ RUN /bin/bash -c "pip3.8 install -r /dh.requirements.txt" \
 
 ADD backend/dataall /dataall
 VOLUME ["/dataall"]
-ADD backend/blueprints /blueprints
 ADD backend/cdkproxymain.py /cdkproxymain.py
 
-RUN mkdir -p dataall/cdkproxy/assets/glueprofilingjob/jars \
-    && mkdir -p blueprints/ml_data_pipeline/engine/glue/jars
+RUN mkdir -p dataall/cdkproxy/assets/glueprofilingjob/jars
```

Comment:
I have noticed an error in the profiling jobs unable to find this jar. I think it is related with this task but we will take it in a separate PR